packages/@webex/media-helpers/package.json

```diff
@@ -28,7 +28,7 @@
     "@webex/jest-config-legacy": "workspace:^",
     "@webex/legacy-tools": "workspace:^",
     "@webex/ts-events": "^1.1.0",
-    "@webex/web-media-effects": "^2.11.5"
+    "@webex/web-media-effects": "^2.12.0"
```

Comment:
@meta-dipanshu-sharma - We've to align with this version in the stream-classes branch as well. Kindly check

Reply:
Yes, @Kesari3008 Could you please update the version to `2.13.1`.

Reply:
Done

Reply:
Thanks @Kesari3008 & @meta-dipanshu-sharma